docs/manual.md

```diff
@@ -103,6 +102,17 @@ Turso aims towards full SQLite compatibility but has the following limitations:
 
 For more detailed list of SQLite compatibility, please refer to [COMPAT.md](../COMPAT.md).
 
+#### MVCC limitations
+
+The MVCC implementation is experimental and has the following limitations:
+
+* Indexes cannot be created and databases with indexes cannot be used.
+* All the data is eagerly loaded to memory at startup so using big databases may take a long time to start, and will consume a lot of memory
```

Comment:
Actually my bad, this should be: `All the data for a table is eagerly loaded from disk to memory when it is first accessed, ...`